curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23f9bf86c76b9248c6eba.md

```diff
@@ -9,6 +9,12 @@ dashedName: step-7
 
 You can add images to your website by using the `img` element. `img` elements have an opening tag without a closing tag. A tag for an element without a closing tag is known as a <dfn>self-closing tag</dfn>.
 
+Here is the basic syntax for an `img` element:
+
+```html
+<img>
+```
```

Comment:
Adding this example flat out gives away the answer.

The tests should be sufficient enough here - campers are familiar with how tags work.

Reply:
This doesn't seem to show up much on the forum, so I can remove the code example